utils/utils.py

```
@@ -325,4 +325,10 @@ def ensure_podman_running(client=None):
         else:
             print("Podman machine started successfully.")
     else:
-        print("Podman is already running.")
\ No newline at end of file
+        print("Podman is already running.")
+
+def normalise_url(version, url):
+    if not url:
+        raise ValueError("url cannot be empty")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)